c2c-extension/src/main/resources/c2c-schema.json

```diff
@@ -236,99 +267,119 @@
           }
         },
         "deployment": {
+          "description": "Contains the properties related to the deployment.",
           "type": "object",
           "additionalProperties": false,
           "properties": {
             "internal_domain_name": {
+              "description": "Name of the internal domain (k8s service name)",
               "type": "string",
               "pattern": "[a-z0-9]([-a-z0-9]*[a-z0-9])?",
               "message": {
                 "pattern": "`internal_domain_name` must start with lowercase alphanumeric character and can then it could contain dash in the middle of the string"
```

Comment:
```suggestion
                "pattern": "The `internal_domain_name` must start with a lowercase alphanumeric character. It can contain a dash in the middle of the string"
```